src/docs/guides/23-how-to-use-custom-pricing/index.md

```diff
@@ -0,0 +1,793 @@
+  unitType={unitType}
+  onSubmit={handleSubmit}
+  price={price}
+  isOwnListing={isOwnListing}
+  timeSlots={timeSlots}
+  fetchTimeSlotsError={fetchTimeSlotsError}
+  cleaningFee={cleaningFee}
+/>
+```
+
+In the `BookingDatesForm` component file import the `FieldCheckbox`
+component and the `formatMoney` utility function:
+
+```js
+import {
+  Form,
+  PrimaryButton,
+  FieldDateRangeInput,
+  FieldCheckbox,
+} from '../../components';
+import { formatMoney } from '../../util/currency';
+```
+
+Add a new entry to the component props map: `cleaningFee`. It holds the
+price of the listing's cleaning fee in a `Money` object. Also we need to
+resolve the current value for an `additionalItems` input in the form and
+a label from the translations so that we can add an input for cleaning
+fee to the form.
+
+To resolve the current cleaning fee selection from the form, add the
+following lines to the `render` funciton of the `FinalForm` element in
+`BookingDatesForm`:
+
+```js
+const selectedCleaningFee =
+  values &&
+  values.additionalItems &&
+  values.additionalItems[0] === 'cleaningFee'
+    ? cleaningFee
+    : null;
+```
+
+Add the resolved `selectedCleaningFee` as a `cleaningFee` attribute to
+the bookingData object:
+
+```js
+const bookingData =
+  startDate && endDate
+    ? {
+        unitType,
+        unitPrice,
+        startDate,
+        endDate,
+
+        // NOTE: If unitType is `line-item/units`, a new picker
+        // for the quantity should be added to the form.
+        quantity: 1,
+        cleaningFee: selectedCleaningFee,
+      }
+    : null;
+```
+
+Also add the following line inside the `BookingDatesForm` component
+before the `return` statement:
+
+```js
+const cleaningFeeLabel = intl.formatMessage({
+  id: 'BookingDatesForm.cleaningFee',
+});
+```
+
+Now we can add an input for selecting a cleaning fee to be added to a
+booking, for example right after the `FieldDateRangeInput` inside the
+`Form` element:
+
+```js
+{
+  cleaningFee ? (
+    <div className={css.cleaningFee}>
+      <FieldCheckbox
+        className={css.cleaningFeeLabel}
+        id={`${form}.cleaningFee`}
+        label={cleaningFeeLabel}
+        name={'additionalItems'}
+        value={'cleaningFee'}
+      />
+      <span className={css.cleaningFeeAmount}>
+        {formatMoney(intl, cleaningFee)}
+      </span>
+    </div>
+  ) : null;
+}
+```
+
+The input can be styled by adding the following style definitions to
+`BookingDatesForm.css`:
+
+```css
+.cleaningFee {
+  margin: 0 24px 38px 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-shrink: 0;
+
+  @media (--viewportMedium) {
+    margin: 0 0 38px 0;
+  }
+}
+
+.cleaningFeeAmount {
+  @apply --marketplaceListingAttributeFontStyles;
+  font-weight: 600;
+  margin: 0 0 0 10px;
+
+  @media (--viewportMedium) {
+    font-weight: 700;
+    margin: 0 0 0 10px;
+  }
+}
+```
+
+The next task is to modify the `EstimatedBreakdownMaybe` component to
```

Comment:
Add a link to the template app since it's not clear that EstimatedBreakdownMaybe subcomponent is inside of src/forms/BookingDatesForm directory.